src/data/loader.js

```diff
@@ -2,7 +2,7 @@ const importer = require.context('./', true, /\.(yml|yaml)/);
 const keys = [...new Set([...importer.keys()])];
 
 module.exports = keys.reduce((prev, cur) => {
-  const fileName = cur.replace(/\.(yaml|yml)/, '');
```

Comment:
Why change this?

Reply:
i add `replace('./','')` so that i can get only the filename, like a11y.yaml instead of ./a11y.yaml